exe/bolt

```diff
@@ -1,6 +1,21 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
+bootsnap_enabled = ENV['BOLT_BOOTSNAP'] || false
+if bootsnap_enabled
```

Comment:
Is this just for testing bootsnap vs. no-bootsnap times? If so, we should remove it before merging. Otherwise, this lgtm.

Reply:
We did this to ship it as an opt-in feature so we can get feedback on if/how it helps. The intention is to eventually make it on by default if it works out well

Reply:
👍 Got it. If we're looking to get feedback from users instead of just internally, we should probably add a section to the docs somewhere describing how to enable bootsnap & send us feedback. @hestonhoffman any thoughts?

Reply:
Great minds think alike. I added a section to the experiment docs for this feature

Reply:
I guess my only other thought is that a post on general Windows performance improvements in the Dev update would be a good place to track the progress on this stuff